generators/app/constants.js

```diff
@@ -45,6 +45,11 @@ module.exports.FLOWCONFIG_PATH = {
   destination: '.flowconfig'
 };
 
+module.exports.BABELRC_PATH = {
+  src: 'babelrc',
+  destination: '.babelrc'
+};
```

Comment:
In a next pr i will change this approach, to avoid create an object like this for new config file